01_fib/fibonacco_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
You should add some tests for the `fib` function directly. Test it against some edge case values
eg:
`fib(0)`
`fib(big number)`